<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>816</width>
    <height>521</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>线性代数计算器</string>
  </property>
  <property name="windowIcon">
   <iconset resource="img_src.qrc">
    <normaloff>:/math.ico</normaloff>:/math.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="baseSize">
    <size>
     <width>600</width>
     <height>800</height>
    </size>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>30</y>
      <width>691</width>
      <height>401</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QPushButton" name="inputmatix">
           <property name="text">
            <string>点击输入矩阵</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="load">
           <property name="text">
            <string>载入内存</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="write">
           <property name="text">
            <string>存入文件</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="text2">
         <property name="text">
          <string>工作区</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="var_display"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="text1">
       <property name="text">
        <string>输入表达式计算：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="expression_input"/>
     </item>
     <item>
      <widget class="QLabel" name="outcome">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="button_clear">
         <property name="text">
          <string>清空</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="button_compute">
         <property name="text">
          <string>计算结果</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>816</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="img_src.qrc"/>
 </resources>
 <connections/>
</ui>
